js/lib/MerkleTree.js

```diff
@@ -53,6 +53,10 @@ class MerkleTree {
     return bufferToHex(this.getRoot());
   }
 
+  getProofByElem (el) {
+      return this.getProof(sha3(el))
```

Comment:
also we use a single tab

Reply:
@lchangdev thought we use two spaces, no tabs?